test-it/karma/fixtures/javascriptinvoke/CloseReference.via

```diff
@@ -0,0 +1,111 @@
+define(DynamicReferencesVI dv(VirtualInstrument (
+    Locals: c(
+        ce(dv(.ErrorCluster (true 100 'error' )) errorIn)
```

Comment:
<nitpick> I would just pick a different string for source instead of 'error'. Too many error(s) already :-)

Reply:
Done.